src/transformers/models/xglm/modeling_tf_xglm.py

```diff
@@ -463,19 +463,12 @@ def _prepare_decoder_attention_mask(
     ) -> tf.Tensor:
         # create causal mask
         # [bsz, seq_len] -> [bsz, 1, tgt_seq_len, src_seq_len]
-        combined_attention_mask: tf.Tensor | None = None
-        if input_shape[-1] > 1:
```

Comment:
we don't need to check this condition anymore ..?

Reply:
+1

Reply:
I don't think so! I couldn't see a case where `input_shape[-1] == 0` was possible.

Reply:
I mean `> 1` vs `== 1` not `== 0`

Reply:
Ah, you're totally right, I don't know how I blanked on that! Let me fix it.